pkg/otlp/config_test.go

```diff
@@ -104,11 +96,74 @@ func TestFromAgentConfigPort(t *testing.T) {
 			path: "port/alldisabled.yaml",
 			err:  "at least one OTLP signal needs to be enabled",
 		},
+		{
```

Comment:
Nit: I'd rename the test to something more generic than `TestFromAgentConfigPort` (maybe `TestFromAgentConfigReceiver`), since it tests more than the port configuration options.

Reply:
Done!